apps/omg_eth/lib/omg_eth/subscription_worker.ex

```diff
@@ -59,13 +59,14 @@ defmodule OMG.Eth.SubscriptionWorker do
     {:ok, opts}
   end
 
+  # sobelow_skip ["DOS.StringToAtom"]
   @doc false
   @impl true
   def handle_frame({:text, msg}, state) do
     {:ok, decoded} = Jason.decode(msg)
     listen_to = Keyword.fetch!(state, :listen_to)
     event_bus = Keyword.fetch!(state, :event_bus)
-    :ok = apply(event_bus, :broadcast, [listen_to, {String.to_existing_atom(listen_to), decoded}])
+    :ok = apply(event_bus, :broadcast, [listen_to, {String.to_atom(listen_to), decoded}])
```

Comment:
Why? I guess this `listen_to` is set during startup, so it isn't potential vulnerability however why it's changed in PR for swagger?

Reply:
A test was broken in one of the runs because there was no atom for this. 
It’s not a vulnerability because the input argument comes from the implementation not from unknown sources.